src/main/java/org/launchcode/techjobs/oo/Job.java

```diff
@@ -17,9 +18,112 @@ public class Job {
     //  other five fields. The second constructor should also call the first in order to initialize
     //  the 'id' field.
 
+    public Job() {
+        id = nextId;
+        nextId++;
+    }
+
+    public Job(String name, Employer employer, Location location, PositionType positionType, CoreCompetency coreCompetency) {
+        this();
+//        if (name.length() < 1) {
+//            name = "Data not available";
+//        }
+//        else {
+//            this.name = name;
+//        }
```

Comment:
```suggestion
        if (name.length() < 1) {
            name = "Data not available";
        }
        else {
            this.name = name;
        }
```

Uncommenting this and your `name` check in your `testJobStringBlank` will makes this test pass!